{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/tools/cli.py

```diff
@@ -6,136 +6,159 @@
-              show_default=True,
+@click.option('-l', '--locations-file',
               type=click.Path(dir_okay=False),
-              help=f'The file with the location parameters for the template.')
+              help=('The file with the location parameters for the template. If no locations file is provided '
+                    'and no single location is provided with the ``-s`` flag, this will default to the locations '
+                    f'file located at {str(MODEL_SPEC_DIR / "locations.txt")}.'))
 @click.option('-t', '--template',
-              default=MODEL_SPEC_DIR / 'model_spec.in',
+              default=str(MODEL_SPEC_DIR / 'model_spec.in'),
               show_default=True,
-              type=click.Path(dir_okay=False),
-              help='The model spec template file')
-@click.option('-s', '--single_location',
+              type=click.Path(exists=True, dir_okay=False),
+              help='The model specification template file.')
+@click.option('-s', '--single-location',
               default='',
-              help='Specify a single location name. This takes precedence over the default locations file.')
-@click.option('-o', '--output_dir',
-              default=MODEL_SPEC_DIR,
+              help='Specify a single location name.')
+@click.option('-o', '--output-dir',
+              default=str(MODEL_SPEC_DIR),
               show_default=True,
-              type=click.Path(dir_okay=True),
+              type=click.Path(exists=True, dir_okay=True),
               help='Specify an output directory. Directory must exist.')
-def make_specs(template: Path, locations_file: Path, single_location: str, output_dir: Path) -> None:
-    """
-    Generate model specifications based on a template. The template lives here:
+def make_specs(template: str, locations_file: str, single_location: str, output_dir: str) -> None:
+    """Generate model specifications based on a template.
 
-    ../{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/model_specification/model_spec.in
+    The default template lives here:
 
-    Supply the locations for which you want a model spec generated by filling in the empty 'locations.txt' file.
-    The location file is in the same directory as the model specification file. It should contain a single location
-    per line and locations must be formatted so that they match location names in GBD location set X.
+    ``{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/model_specification/model_spec.in``
 
-    Valid location sets can be found in:
+    Supply the locations for which you want a model spec generated by filling
+    in the empty 'locations.txt' file. A template for this file can be found at
 
-    https://stash.ihme.washington.edu/projects/CSTE/repos/vivarium_gbd_access/browse/src/vivarium_gbd_access/gbd.py
+    ``{{cookiecutter.package_name}}/src/{{cookiecutter.package_name}}/model_specification/locations.txt``
```

Comment:
In general, referencing specific external code in comments is super fragile.  We're about to do major updates to gbd access and I'd definitely forget to come update this comment.